tests/integration/test_api_gateway.py

```diff
@@ -360,7 +362,79 @@ def test_api_gateway_authorizer_crud(self):
             Exception,
             apig.get_authorizer,
             self.TEST_API_GATEWAY_ID,
-            authorizer_id)
+            authorizer_id
+        )
+
+    def test_apigateway_with_lambda_integration(self):
+        self.create_lambda_function(TEST_API_GATEWAY_INTEGRATION_LAMBDA_FUNCTION_NAME)
+
+        lambda_uri = aws_stack.lambda_function_arn(TEST_API_GATEWAY_INTEGRATION_LAMBDA_FUNCTION_NAME)
+        invocation_uri = 'arn:aws:apigateway:%s:lambda:path/2015-03-31/functions/%s/invocations'
```

Comment:
nit: We could use `apigateway_invocations_arn(...)` from `utils/aws/aws_stack.py` here.